src/FESpaces/FESpaceFactories.jl

```diff
@@ -64,19 +64,21 @@ end
 
 function FESpace(
   t::Triangulation,
-  cell_reffe::AbstractArray{<:ReferenceFE};
+  cell_reffe::AbstractArray{<:ReferenceFE},
```

Comment:
Based on my comment above, I would revert all changes from here to the end of the file